internal/api_admin/server.go

```diff
@@ -323,6 +328,81 @@ func (s *Server) RevokeConnectionCredentials(ctx context.Context, request Revoke
 	return RevokeConnectionCredentials202JSONResponse{Message: "Credentials revocation request sent"}, nil
 }
 
+// CreateLink - creates a link for issuing a credential
+func (s *Server) CreateLink(ctx context.Context, request CreateLinkRequestObject) (CreateLinkResponseObject, error) {
+	var expiresAt *time.Time
+	if request.Body.ExpirationDate != nil {
+		layout := "2006/01/02"
+		t, err := time.Parse(layout, *request.Body.ExpirationDate)
+		expiresAt = &t
+		if err != nil {
+			return CreateLink400JSONResponse{N400JSONResponse{Message: "wrong expirationDate attribute format"}}, nil
+		}
+		if isBeforeTomorrow(t) {
+			return CreateLink400JSONResponse{N400JSONResponse{Message: "expirationDate cannot be a date before tomorrow"}}, nil
+		}
+	}
+
+	if request.Body.ClaimLinkExpiration != nil {
+		if !isClaimLinkValid(*request.Body.ClaimLinkExpiration, expiresAt) {
```

Comment:
This should be changed as well

Reply:
Can `ClaimLinkExpiration` be set before today?